src/vs/editor/common/controller/cursor.ts

```diff
@@ -1092,7 +1093,8 @@ export class Cursor extends EventEmitter {
 				postOperationRunnable: null,
 				shouldPushStackElementBefore: false,
 				shouldPushStackElementAfter: false,
-				requestScrollDeltaLines: 0
+				requestScrollDeltaLines: 0,
+				eventData: ctx.eventData
```

Comment:
Since arguments to the command are same for all cursors, I am directly passing them to the OneCursor method instead of passing to the context.